__tests__/app.test.js

```diff
@@ -41,7 +41,34 @@ describe("CORE: GET /api", () => {
       .expect(200)
       .then((response) => {
         const { endpoints } = response.body;
-        expect(endpoints).toBe(endpoints);
+        expect(endpoints).toEqual(endpointsFile);
+      });
+  });
+});
+
+describe("CORE: GET /api/articles/:article_id", () => {
+  test("should return an article object, which should have the article properties", () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.article.article_id).toBe(4);
```

Comment:
I would prefer it if we asserted against all the properties we expect to see returned here on the specific article